roles/confluent.variables/vars/main.yml

```diff
@@ -1064,3 +1064,15 @@ control_center_final_properties: "{{ control_center_combined_properties | combin
 
 ### The base path for the binary files. When in Archive File deployment mode this results in binary files being based in, for example `/opt/confluent/confluent-5.5.1/bin`, otherwise they are based in `/usr/bin`.
 binary_base_path: "{{ archive_config_base_path+'/confluent-'+confluent_package_version if installation_method == 'archive' else '/usr' }}"
+
```

Comment:
Can you move these into `confluent.common/vars/main.yml` ? This file is already super long and I think these are only used by common role